feat(documentation): add styles getting started page (#1468)
swisspost/design-system: packages/documentation/src/stories/get-started/styles.stories.mdx

import { Meta } from '@storybook/addon-docs';
import { BADGE } from '@geometricpanda/storybook-addon-badges';

<Meta
  title="Get Started/Styles"
  parameters={{
    previewTabs: {
      canvas: {
        hidden: true,
      },
    },
    viewMode: 'docs',
    badges: [BADGE.NEEDS_REVISION],
  }}
/>

> # @swisspost/design-system-styles

Reviewer: ```suggestion
<h1>@swisspost/design-system-styles<br><span class="fw-light">Styles for the Swiss Post web platform.</span></h1>
```

> <h1>@swisspost/design-system-styles<br><span class="fw-light">Styles for the Swiss Post web platform.</span></h1>
> 
> [![npm version](https://badge.fury.io/js/@swisspost%2Fdesign-system-styles.svg)](https://badge.fury.io/js/@swisspost%2Fdesign-system-styles)

Reviewer: Broke the heading...

```suggestion
[![npm version](https://badge.fury.io/js/@swisspost%2Fdesign-system-styles.svg)](https://badge.fury.io/js/@swisspost%2Fdesign-system-styles)

# @swisspost/design-system-styles

CSS Styles for the Swiss Post web platform.
```

Author: Have not seen it either!

## Installation

```bash dark
  npm install @swisspost/design-system-styles
```

## Usage

Import one of our stylesheets into your project (for example into your `/src/styles.scss` file).

```scss dark
@use '@swisspost/design-system-styles/<bundle-name>.<extension>';
```

### Available bundles

| Bundle name | Bundle extensions | Description                                                        |
| :---------- | :---------------- | :----------------------------------------------------------------- |
| `index`     | `css`, `scss`     | everything for internet facing applications                        |
| `intranet`  | `css`, `scss`     | everything for internal applications                               |

> | `basics`    | `css`, `scss`     | atomic styles for font, buttons, lists, etc. (no component styles) |

Reviewer: ```suggestion
| `basics`    | `css`, `scss`     | atomic styles for font, buttons, lists, etc. (no component styles) |
| `core`      | `scss`              | variables, mixins, functions and placeholders (no CSS output) |
```

Author: The core is only used to write custom styles and therefore no entry file.
That's why I described the usage of it in its own subsection.

> ### The core

Reviewer: ```suggestion
### Variables, mixins, functions and placeholders
```

> If you want to use our SCSS core for your custom styles, we recommend to import and use it as follows, to prevent conflicts:

Reviewer: ```suggestion
Use the SCSS core for your custom styles and make sure you're always using the most up to date definitions:
```

<p class="alert alert-info alert-md">
  NOTE: The core file itself, does not produce any CSS output. It only contains variables, functions
  and mixins.
</p>

```scss dark
@use '@swisspost/design-system-styles/core.scss' as post;

.my-component {

>   background-color: post.$white;

Reviewer: ```suggestion
  background-color: post.$yellow; // #FFCC00
```
